x/supply/alias.go

```diff
@@ -0,0 +1,28 @@
+// nolint
```

Comment:
Why do we need a whole module for this? We can use the default supply module instead

Reply:
Hmmm we can keep x/supply as small as possible, but lots of variables imported from cosmos supply in app.go will have to be renamed from `supply.xxx` to `cosmosSupply.xxx` if we don't have this alias in x/supply.

Reply:
Hmmm we can keep x/supply as small as possible, but lots of variables imported from cosmos supply in app.go will have to be renamed from supply.xxx to cosmosSupply.xxx if we don't have this alias in x/supply.

Reply:
Oh the reason we have this x/supply is that we need to change the denom used in cosmos supply's simulation from default `stake` to our `uctk`.

Reply:
we can use `stake` denom for simulations. the denom doesn't matter.

Reply:
But in x/oracle we explicitly ask for `uctk` 🤣 does this need to be changed?